discord/errors.py

```diff
@@ -53,6 +53,12 @@
     'ConnectionClosed',
     'PrivilegedIntentsRequired',
     'InteractionResponded',
+    'ExtensionError',
+    'ExtensionAlreadyLoaded',
+    'ExtensionNotLoaded',
+    'NoEntryPointError',
+    'ExtensionFailed',
+    'ExtensionNotFound'
```

Comment:
These aren't really Discord errors IMO. I support moving them out of `ext.commands`, but something more cog specific might be best.